README.md

```diff
@@ -1,378 +1,353 @@
+
+When a command is invoked, `commandHandler.js` checks permissions and other options regarding the command, and then calls the associated function for the command.
+
+#### How to create a command <a name="how-to-create-a-command"></a>
+Commands are made using the [slashCommandBuilder](https://v13.discordjs.guide/popular-topics/builders.html#commands)
+Check out  `./commands/example/exampleCommand.js` for an example.
+1. Under `/commands`, make a folder for the new command or place it in an existing folder that makes sense.
+
+2. Add a new file to that folder, name is `yourCommandNameHere.js`
+3.  The command file has to export a `command` object. There a at least 3 main components of a command. `data`, `options` and `execute`.
+4.  `data` represents the   data of the command, specifically the `slashCommandBuilder`. It contains the name, description, options and permissions of the command. 
+	```js
+	data:  new  SlashCommandBuilder()
+	.setName('example')
+	.setDescription('test')
+	.addStringOption(option =>
+		option.setName('example-option)
+		options.setDescription('this is an example)
+	),
+5. `options` contains options for the command. `deleted` specifies to not place this command on the server. `devOnly` specifies developers are only allowed to run the command. `testOnly` specifies the command can only be run in a specific test server.  **Note:** `options` is optional and all of the variables inside will default to `false` if not provided.
+
+	```js	
+	options:
+	{
+		deleted:  false,
+		devOnly:  true,
+		testOnly:  false,
+	},
+	```
+6. `execute` contains the callback function that is run when the command is called. Place the commands execution logic in here. A good pattern to follow is to make a helper function for the command logic, then inside the execute, grab the needed data and call the helper. 
+	```js
+	async execute(client, interaction){
+		interaction.reply('hello');
+		//or
+		interaction.deferReply();
+		interaction.editReply('Hello);
+	}
+	```
+7. `client` and `interaction` are required parameters for the function. `client` refers to the bot itself, while `interaction` is the data behind the command (such as channel, user, other options...). `client` does not need to be used, but `interaction` does have to be replied to. Examples of such exist in every command.
+
+#### How to create a subcommand <a name="how-to-create-a-subcommand"></a>
+1. Make a command following the above instructions. Once again look at `/commands/example/exampleCommand.js`
+
+3. Add a subcommand via `.addSubCommand`.  `subCommands` and options such as `addStringOption` are mutually exclusive. You cannot have both on a main command. Subcommands can still have options. Set up a name, description and options for each subcommand created.
```

Comment:
The "mutually exclusive" wording here is a bit confusing. Maybe something like "A slash command with subcommands cannot have its own options or executable functions"

Reply:
@wct9667 Would you be able to rewrite this? I'm not fully sure what is being said here